discovery-provider/requirements.txt

```diff
@@ -14,4 +14,5 @@ pytest-cov==2.6.0
 base58==1.0.0
 flask-cors==3.0.6
 gunicorn==19.9.0
-jsonschema==3.2.0
\ No newline at end of file
+jsonschema==3.2.0
+eth_account==0.5.2
```

Comment:
this was never added so alembic wouldn't run locally (b/c we import something from api_helpers which depends on this)

Reply:
huh weird like alembic never ran at all? including local init of web server

Reply:
I'm really not sure, i get this
```
ImportError: cannot import name 'encode_defunct' from 'eth_account.messages
```

Adding to requirements.txt fixes this, which makes sense. i have no idea how any service has been running w/ this tho??

Reply:
@vicky-g noticed this locally i think? idk i'm not clear how we are running without your fix in this pr